packages/contracts/contracts/optimistic-ethereum/ovm/SafetyChecker.sol

```diff
@@ -63,136 +35,102 @@ contract SafetyChecker is ContractResolver {
         view
         returns (bool)
     {
-        bool seenJUMP = false;
-        bool insideUnreachableCode = false;
-        uint256 ops = 0;
-        for (uint256 pc = 0; pc < _bytecode.length; pc++) {
+        // autogenerated by gen_safety_checker_constants.py
+        // number of bytes to skip for each opcode
+        uint256[8] memory skip = [
+          uint256(0x0001010101010101010101010000000001010101010101010101010101010000),
+          uint256(0x0100000000000000000000000000000000000000010101010101000000010100),
+          uint256(0x0000000000000000000000000000000001010101000000010101010100000000),
+          uint256(0x0203040500000000000000000000000000000000000000000000000000000000),
+          uint256(0x0101010101010101010101010101010101010101010101010101010101010101),
+          uint256(0x0101010101000000000000000000000000000000000000000000000000000000),
+          uint256(0x0000000000000000000000000000000000000000000000000000000000000000),
+          uint256(0x0000000000000000000000000000000000000000000000000000000000000000)];
+        // Mask to gate opcode specific cases
+        uint256 _opcodeProcMask = ~uint256(0xffffffffffffffffffffffe000000000fffffffff070ffff9c0ffffec000f001);
+        // Halting opcodes
+        uint256 _opcodeStopMask = ~uint256(0x6008000000000000000000000000000000000000004000000000000000000001);
```

Comment:
since `stop` is an actual opcode would rename this to something like `opcodeStartUnreachableMask`